setup.py

```diff
@@ -52,7 +52,7 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-# PACKAGE_DATA = {'pydicom': ['../tests/tests_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
Test data was inside the package years ago, and I deliberately kept the size small, downsizing images where possible, to keep package size down and to make it easier to work with examples.  Somewhere along the way it got moved out in some restructuring.